angry-bot.js

```diff
@@ -162,6 +162,14 @@ client.on("message", msg => {
         StatHandler.incrementStat(StatHandler.DIVOTKEY_REACTIONS);
     }
 
+    // Medientheorie
+    if (
+        msg.cleanContent.toLocaleLowerCase().includes("medien") &&
+        msg.cleanContent.toLocaleLowerCase().includes("theorie") 
+    ) {
+        msg.reply(`MEDIENTHEORIE :heart_eyes::heart_eyes::heart_eyes:`);
```

Comment:
I weiß grad nimma auswendig ob des a Problem is wenn de Emojis direkt nebeneinander san aber i glaub es sollte passen 👍🏼